core/arch/arm/include/tee/entry_std.h

```diff
@@ -11,5 +11,6 @@
 
 /* Standard call entry */
 void tee_entry_std(struct thread_smc_args *args);
+TEE_Result nsec_sessions_list_head(struct tee_ta_session_head **open_sessions);
```

Comment:
Add an empty space before and a short comment, as `/* Get list head for sessions opened from non-secure */`
As this function cannot fail, you can remove the return code.